PWGGA/GammaConvBase/AliConversionMesonCuts.cxx

```diff
@@ -4592,19 +4592,29 @@ Bool_t AliConversionMesonCuts::MesonIsSelectedByMassCut(AliAODConversionMother *
       }
   } else if (fUsePtDepSelectionWindow){
       // Determine correct mass parametrisation depending on what method is used
-      Float_t pt   = meson->Pt();
+      Float_t pt        = meson->Pt();
+      Float_t pt_used;
       Float_t mass = 0;
       Float_t sigma = 999;
       switch(fMassParamFunction){
         case 0: // EMC-EMC
-          if ( pt>45. ){
-              Double_t pt_fix=45.;
-              mass = (0.125281) + ((0.0023329) * pt_fix) + ((-0.000272107) * pt_fix * pt_fix) + ((1.57169e-05) * pt_fix * pt_fix * pt_fix) + ((-1.93425e-07) * pt_fix * pt_fix * pt_fix * pt_fix);
-              sigma = (0.0208639) + ((-0.00227399) * pt_fix ) + ((0.000162993) * pt_fix * pt_fix) + ((-1.94142e-06) * pt_fix * pt_fix * pt_fix);
+          if (pt<2.0){
```

Comment:
isn't pt_used = pt in all three cases?
so maybe just omit the ifs and else?

Reply:
correct